src/main/java/com/dev/cinema/controllers/CinemaHallController.java

```diff
@@ -0,0 +1,47 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.model.dto.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.CinemaHallResponseDto;
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.service.CinemaHallService;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author Sergey Klunniy
+ */
+@RestController
+@RequestMapping("/cinemaHall")
```

Comment:
```suggestion
@RequestMapping("/cinema-halls")
```
https://restfulapi.net/resource-naming/

Reply:
я виправив на cinema-halls